Unix/wsman/wsmanclient.c

```diff
@@ -378,12 +383,14 @@ static MI_Boolean ProcessFaultResponse(WsmanClient *self, Page **data)
             errorMessage = fault.mi_message;
         }
 
+        omiErrorPtr = GetWsmanCimError(errorType, &omiError);
+
         if (self->enumerationState)
         {
             self->enumerationState->endOfSequence = 1;
         }
 
-        PostResult(self, errorMessage, fault.mi_result > 0 ? fault.mi_result : MI_RESULT_FAILED);
+        PostResult(self, errorMessage, fault.mi_result > 0 ? fault.mi_result : MI_RESULT_FAILED, omiErrorPtr);
```

Comment:
I believe this is a Schedule post on the strand which means there is a chance a different thread could pick it up and deliver to the C API. As a result omiError has gone out of scope from the stack.
Also, the instance was not Constructed (something like InstanceConstruct that creates it in place)